packages/components/src/list/collapsible-list/index.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { Dashicon } from '@wordpress/components';
+import { useState, useCallback, Children } from '@wordpress/element';
+import { Transition } from 'react-transition-group';
+
+/**
+ * Internal dependencies
+ */
+import { ExperimentalListItem } from '../experimental-list-item';
+import { ListProps, ExperimentalList } from '../experimental-list';
+
+type CollapsibleListProps = {
+	hideText: string;
```

Comment:
I'm starting to scrutinize naming, sorry 🙃 ..

How about `collapseLabel` or `collapseButtonLabel`?

Just trying to make this match up to `onCollapse` and `onExpand`.

Reply:
All good, this makes sense, done ✅